README.md

```diff
@@ -30,5 +30,372 @@
+  supported. Triton also supports TensorFlow-TensorRT, ONNX-TensorRT
+  and PyTorch-TensorRT integrated models.
+
+* [Machine learning
+  frameworks](https://github.com/triton-inference-server/fil_backend).
+  Triton supports popular machine learning frameworks such as XGBoost,
+  LightGBM, Scikit-Learn and cuML using the [RAPIDS Forest Inference
+  Library](https://medium.com/rapids-ai/rapids-forest-inference-library-prediction-at-100-million-rows-per-second-19558890bc35).
+
+* [Concurrent model
+  execution](docs/architecture.md#concurrent-model-execution). Triton
+  can simultaneously run multiple models (or multiple instances of the
+  same model) using the same or different deep-learning and
+  machine-learning frameworks.
+
+* [Dynamic batching](docs/architecture.md#models-and-schedulers). For
+  models that support batching, Triton implements multiple scheduling
+  and batching algorithms that combine individual inference requests
+  together to improve inference throughput. These scheduling and
+  batching decisions are transparent to the client requesting
+  inference.
+
+* [Extensible
+  backends](https://github.com/triton-inference-server/backend). In
+  addition to deep-learning frameworks, Triton provides a *backend
+  API* that allows Triton to be extended with any model execution
+  logic implemented in
+  [Python](https://github.com/triton-inference-server/python_backend)
+  or
+  [C++](https://github.com/triton-inference-server/backend/blob/main/README.md#triton-backend-api),
+  while still benefiting from full CPU and GPU support, concurrent
+  execution, dynamic batching and other features provided by Triton.
+
+* Model pipelines using
+  [Ensembling](docs/architecture.md#ensemble-models) or [Business
+  Logic Scripting
+  (BLS)](https://github.com/triton-inference-server/python_backend#business-logic-scripting).
+  A Triton *ensemble* represents a pipeline of one or more models and
+  the connection of input and output tensors between those
+  models. *BLS* allows a pipeline along with extra business logic to
+  be represented in Python. In both cases a single inference request
+  will trigger the execution of the entire pipeline.
+
+* [HTTP/REST and GRPC inference
+  protocols](docs/inference_protocols.md) based on the community
+  developed [KServe
+  protocol](https://github.com/kserve/kserve/tree/master/docs/predict-api/v2).
+
+* A [C API](docs/inference_protocols.md#in-process-triton-server-api)
+  allows Triton to be linked directly into your application for edge
+  and other in-process use cases.
+
+* A [Java API](docs/inference_protocols.md#java-api) is similar to C API and
+  allows Triton to be linked directly to your Java application.
+
+* [Metrics](docs/metrics.md) indicating GPU utilization, server
+  throughput, and server latency. The metrics are provided in
+  Prometheus data format.
+
+## Documentation
+
+**The master branch documentation tracks the upcoming,
```

Comment:
Remove this paragraph for release branches

Reply:
@dzier - I've removed mentioned paragraph